### PR TITLE
Merge 1.10.x into master

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
Merge 1.10.x into master

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/user-guide/cloud-storage.rst [hunk 1]
```diff
@@ -43,8 +45,8 @@ Rasa supports using `S3 <https://aws.amazon.com/s3/>`_ ,
     variable to the path of that key file.
 
 * Azure Storage
-    Azure is supported using the legacy ``azure-storage-blob`` package (v 2.1.0),
-    which you can install with ``pip install -I azure-storage-blob==2.1.0``.
+    Azure is supported using the ``azure-storage-blob`` package,
+    which you can install with ``pip install azure-storage-blob``.
```
Comment (tabergma) [inline docs/user-guide/cloud-storage.rst:49]: Not sure which version is the latest one. As @erohmensing is offline, maybe @melindaloubser1 can help?

Comment (melindaloubser1) [reply docs/user-guide/cloud-storage.rst:49]: Ooh I do not know this off the top of my head, but editing times show that the one in master was edited 2 months ago vs the 1.10.x one a year ago.

Comment (melindaloubser1) [reply docs/user-guide/cloud-storage.rst:49]: Did you mean the actual version of azure-storage-blob?

Comment (tabergma) [reply docs/user-guide/cloud-storage.rst:49]: > Did you mean the actual version of azure-storage-blob?

Yeah the version was added on master, but some other changes were made on 1.10.x. 

I guess, I'll keep the master version then. Thanks.

**Comment by tabergma** [comment]
@melindaloubser1 Can you please give this a quick review as you also worked on some of the changes? Thanks.

**Pull request merged**